### ISSUE TITLE
Add support for game Tale of Two Wastelands - using FNV Epic release

### ISSUE THREAD

**Issue by WilliamImm** [ModOrganizer2/modorganizer#1891]
- **Name:** Tale of Two Wastelands, using Fallout: New Vegas (Epic Games Store version)
- **Short name:** ttw
- **Nexus ID [optional]:** https://www.nexusmods.com/newvegas
- **Executable:** FallloutNV.exe
- **Launcher [optional]:** Two: FalloutNVLauncher.exe (usual), or ../Launcher.exe (appears to be a passthrough)
- **Data path:** Data
- **Documents path:** My Games\FalloutNV_Epic
- **Saves directory [optional]:** My  Games\FalloutNV_Epic\Saves
- **Save extension [optional]:** .fos
- **Epic Games Manifests:**
  - Basegame: `5daeb974a22a435988892319b3a4f476`
  - Dead Money: `b290229eb58045cbab9501640f3278f3`
  - Honest Hearts: `562d4a2c1b3147b089a7c453e3ddbcbe`
  - Old World Blues: `c8dae1ab0570475a8b38a9041e614840`
  - Lonesome Road: `4fa3d8d9b2cb4714a19a38d1a598be8f`
  - Gun Runners' Arsenal: `7dcfb9cd9d134728b2646466c34c7b3b`
  - Courier's Stash: `ee9a44b4530942499ef1c8c390731fce`
  
## Additional Details

Basically the changes for #1885, but port to the TTW plugin. Completely forgot that this is technically a separate thing.

Virtualize both %localAppData%\FalloutNV and %localAppData%\FalloutNV_Epic for this release to work around a EGS-introduced bug that prevents plugins from being loaded.

Probably make this inherent from the game_gamebryo detection logic too.

**Comment by WilliamImm** [comment]
Pinging @Silarn for this

**Comment by WilliamImm** [comment]
Closing, as addressed in ModOrganizer2/modorganizer-game_ttw#27
